openpype/hosts/flame/api/plugin.py

```diff
@@ -749,10 +765,37 @@ def _generate_media_info_file(self):
         # execute creation of clip xml template data
         try:
             openpype.run_subprocess(cmd_args)
+            self._make_single_clip_media_info()
         except TypeError:
             self.log.error("Error creating self.tmp_file")
             six.reraise(*sys.exc_info())
 
+    def _make_single_clip_media_info(self):
+        with open(self.tmp_file) as f:
+            lines = f.readlines()
+            _added_root = itertools.chain(
+                "<root>", deepcopy(lines)[1:], "</root>")
+            new_root = ET.fromstringlist(_added_root)
```

Comment:
You don't need to have opened file for the whole time.
```suggestion
        with open(self.tmp_file) as f:
            lines = f.readlines()
        _added_root = itertools.chain(
            "<root>", deepcopy(lines)[1:], "</root>")
        new_root = ET.fromstringlist(_added_root)
```